clippy.toml

```diff
@@ -1 +1,21 @@
+# Project configuration for Clippy Rust code linter.
+# See full lints list at:
+# https://rust-lang.github.io/rust-clippy/master/index.html
+
 msrv = "1.65.0"
+standard-macro-braces = [
```

Comment:
Could use a comment on what this is achieving as opposed to the default settings.

Reply:
After checking the PR more I realize it's to add rules for `quote` and `parse_quote` mostly. But afaict there's no need to redefine the default ones already provided by clippy, because it will add any user configured ones to the default list: https://github.com/rust-lang/rust-clippy/blob/master/clippy_lints/src/nonstandard_macro_braces.rs#L187-L190

Reply:
@JelteF I've removed default ones from the list, and added a comment with the link to the default list: https://github.com/JelteF/derive_more/pull/226/commits/e4fe06551b366ceac3e1de8dbd94a2ef02c895f8